packages/web-components/src/switch/switch.stories.ts

```diff
@@ -0,0 +1,77 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Switch as FluentSwitch } from './switch.js';
+import './define.js';
+import { SwitchLabelPosition } from './switch.options.js';
+
+type SwitchStoryArgs = Args & FluentSwitch;
+type SwitchStoryMeta = Meta<SwitchStoryArgs>;
+
+const storyTemplate = html<SwitchStoryArgs>`
+  <div>
+    <fluent-switch
+      ?checked=${x => x.checked}
+      ?disabled=${x => x.disabled}
+      ?required=${x => x.required}
```

Comment:
In FUIR9, `required` renders an additional star:
![image](https://user-images.githubusercontent.com/9615899/216022309-903666b4-9b0c-4b27-a249-d25b392678a2.png)
https://react.fluentui.dev/?path=/docs/components-switch--default#required